control/statefbk.py

```diff
@@ -721,6 +721,23 @@ def create_statefbk_iosystem(
         System name. If unspecified, a generic name <sys[id]> is generated
         with a unique integer id.
 
+    Examples
+    --------
+    >>> import control as ct
+    >>> import numpy as np
+    >>>
```

Comment:
`np` and `ct` are globally implicit

https://github.com/python-control/python-control/blob/42c6fb1498aa9f5d542113da288c7c4e3547c117/doc/conf.py#L280-L282